crates/bitwarden/src/client/mod.rs

```diff
@@ -4,6 +4,7 @@ pub(crate) use client::*;
 pub(crate) mod access_token;
 #[cfg(any(feature = "internal", feature = "mobile"))]
 pub(crate) mod auth_settings;
+#[allow(clippy::module_inception)]
```

Comment:
Is it best practice for us to look into renaming these files?

Reply:
This seems to be meant to avoid beginners accidentally defining the module twice by not fully understanding the module system, at least according to the lint help: https://rust-lang.github.io/rust-clippy/master/index.html#/module_inception

The `client` module there is private, so I think it's not a huge issue, but if we can come up with a better name that should be preferable IMO.